.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
This file doesn't need to be here; it is just for your local deployment. I recommend deleting it